CHANGELOG.md

```diff
@@ -8,26 +8,31 @@ A note on future revisions.
   Everything within a major version number should be code compatible (With the exception of experimental interfaces).  The most notable example of an experimental interface is the support for multiple source inputs.  The API's to deal with this will change in future minor releases.    Everything within a single minor release should be network compatible with other federates on the same minor release number.  Compatibility across minor release numbers may be possible in some situations but we are not going to guarantee this as those components are subject to performance improvements and may need to be modified at some point.  patch releases will be limited to bug fixes and other improvements not impacting the public API or network compatibility.  Check [here](./docs/Public_API.md) for details of what is included and excluded from the public API and version stability.
 
 ## [2.1.0] - 2019-06-11
-The main focus of this minor release is cleaning up the build system and extracting required compiled libraries from the HELICS build process, no changes in the C api, and few additionals and deprecations in the C++ API related to command line arguments.  
+The main focus of this minor release is cleaning up the build system and extracting required compiled libraries from the HELICS build process, no changes in the C api, and few additions and deprecations in the C++ API related to command line arguments.  
 ### Changed
    - remove use of boost::program options and replace usage with CLI11
    - remove boost::asio and replace with a submodule for ASIO
    - remove included fmt code and replace with submodule
-   - remove jsoncpp code and replace with a submodule which generates a compiled library- this removed the need to continually regenerate the single header/file with customized namespaces, though if you are manually including helics the HELICS::jsoncpp needs to be added as a library.
+   - remove jsoncpp code and replace with a submodule which generates a compiled library- this removed the need to continually regenerate the single header/file with customized namespaces, though if you are manually including helics the HELICS::jsoncpp needs to be added as a library.  Also included is an option to incorporate Jsoncpp as an object library, and create a target HELICS::jsoncpp_headers.
```

Comment:
The part about  manually including the HELICS::jsoncpp is a bit confusing -- is it referring to linking?

```suggestion
   - remove JsonCpp code and replace with a submodule which generates a compiled library - this removed the need to continually regenerate the single header/file with customized namespaces, though if you are using the helics-static library built with a separate JsonCpp static library, the HELICS copy of the jsoncpp static library must be linked with manually (for build systems other than CMake such as waf, meson, premake, etc).  Also included is an option to incorporate JsonCpp as an object library within a single helics-static library (default on macOS/Linux), and create a target HELICS::jsoncpp_headers.
```